src/mapper/pkg/kubefinder/kubefinder.go

```diff
@@ -105,7 +124,9 @@ func (k *KubeFinder) initIndexes(ctx context.Context) error {
 		if svc.DeletionTimestamp != nil {
 			return nil
 		}
-		if svc.Spec.Type != corev1.ServiceTypeNodePort {
+
+		// Node ports are unique regardless of the service type
```

Comment:
should say "Node ports & load balancers are unique ..."?

Reply:
No, but I will split it into two different comments. 
One on the top of this function: "// node ports are unique per service - so it can be used for indexing services", 
and the other for this line: "// Only node port and load balancer typed services use node ports"